### PR TITLE
Drive

### PR DESCRIPTION
**Pull request by kishan243**

### CODE REVIEW

**Comment by kishan243** [comment]
Drive pull request to main

File: src/main/java/org/sciborgs1155/robot/drive/TalonSwerveModule.java [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    turnMotor.setInverted(false);
+    turnMotor.setIdleMode(IdleMode.kBrake);
+    turnMotor.setOpenLoopRampRate(0);
+    turnMotor.setSmartCurrentLimit(20);
+
+    turnEncoder.setInverted(Turning.ENCODER_INVERTED);
+
+    turnEncoder.setPositionConversionFactor(Turning.POSITION_FACTOR.in(Radians));
+    turnEncoder.setVelocityConversionFactor(Turning.POSITION_FACTOR.in(Radians));
+
```
Comment (AngleSideAngle) [inline src/main/java/org/sciborgs1155/robot/drive/TalonSwerveModule.java:57]: The turn motor configuration should be changed to be the same as FlexModule

File: src/main/java/org/sciborgs1155/robot/drive/TalonSwerveModule.java [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+import com.ctre.phoenix6.signals.NeutralModeValue;
+import com.revrobotics.CANSparkBase.IdleMode;
+import com.revrobotics.CANSparkLowLevel.MotorType;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.SparkAbsoluteEncoder;
+import com.revrobotics.SparkAbsoluteEncoder.Type;
+import edu.wpi.first.math.geometry.Rotation2d;
+import org.sciborgs1155.robot.drive.DriveConstants.ModuleConstants.Turning;
+
+public class TalonSwerveModule implements ModuleIO {
```
Comment (AngleSideAngle) [inline src/main/java/org/sciborgs1155/robot/drive/TalonSwerveModule.java:17]: For naming consistency with `FlexModule`, this should be named `TalonModule`

Comment (AngleSideAngle) [reply src/main/java/org/sciborgs1155/robot/drive/TalonSwerveModule.java:17]: Also maybe add a line of javadoc describing what the class is

**Review by AngleSideAngle — CHANGES_REQUESTED** [review]

**Pull request merged**